src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const results = people.filter((data) =>
+    !century ? data.sex === 'm'
+      : data.sex === 'm' && Math.ceil(data.died / 100) === century);
```

Comment:
```suggestion
    !century 
      ? data.sex === 'm'
      : data.sex === 'm' && Math.ceil(data.died / 100) === century);
```
Also, it's better to check if century is present, not its absence